frontend/src/components/SamplesList.tsx

```diff
@@ -117,10 +56,22 @@ export const SamplesList: FunctionComponent<ISampleListProps> = ({
           : {
               first: max_rows,
             }),
-        options: {
+        sampleMetadataOptions: {
           sort: [{ importDate: SortDirection.Desc }],
           limit: 1,
         },
+        bamCompletesOptions: {
+          sort: [{ date: SortDirection.Desc }],
+          limit: 1,
+        },
+        mafCompletesOptions: {
+          sort: [{ date: SortDirection.Desc }],
+          limit: 1,
+        },
+        qcCompletesOptions: {
+          sort: [{ date: SortDirection.Desc }],
+          limit: 1,
+        },
```

Comment:
These arguments let us surface only the latest Tempo event data in the Cohorts page's Samples popups.

(They all have the same content but unfortunately it doesn't appear that GraphQL query allows reusing a single argument input for different variable types. For example, `bamCompletesOptions` is of the type `BamCompletesOptions` and can't be passed into `mafCompletesOptions` of the type `MafCompletesOptions`. Creating a union type doesn't seem to work.)